### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by kikirizki** [nii-yamagishilab/ssnt-tts#1]
Hi Thankyou for great Works, I wonder if you provide pre-trained model? and how long it will take if trained from scratch

**Comment by TanUkkii007** [comment]
Thank you for your interest. It took 9 days to train with batch size 48 on Tesla V100 GPU.
Our model used proprietary labels so I am afraid it is not possible to publish our pre-trained models used in our paper.

**Comment by kikirizki** [comment]
Hi, Thank you for your response, it is okay, I do understand  about that matter.
